config/pipelines/pacbio.yml

```diff
@@ -291,108 +291,33 @@ default: &default
                   type: :model
                   value: bio_sample_name
       v13_revio:
```

Comment:
Can we move the samples section to the RunCsv as well? And remove the need for this?

Reply:
If we keep the previous config here it would allow us to feature flag it which might be useful.